packages/router/package.json

```diff
@@ -0,0 +1,40 @@
+{
+	"name": "@wordpress/router",
+	"version": "0.1.0",
+	"description": "Router API for WordPress pages.",
+	"author": "The WordPress Contributors",
+	"license": "GPL-2.0-or-later",
+	"keywords": [
+		"wordpress",
+		"gutenberg",
+		"router"
+	],
+	"homepage": "https://github.com/WordPress/gutenberg/tree/HEAD/packages/router/README.md",
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/WordPress/gutenberg.git",
+		"directory": "packages/router"
+	},
+	"bugs": {
+		"url": "https://github.com/WordPress/gutenberg/issues"
+	},
+	"engines": {
+		"node": ">=12"
+	},
+	"main": "build/index.js",
+	"module": "build-module/index.js",
+	"react-native": "src/index",
+	"dependencies": {
+		"@babel/runtime": "^7.16.0",
+		"@wordpress/element": "file:../element",
```

Comment:
Slightly off-topic: I know we have precedence but I think it makes more sense to mark `@wordpress/element` as a peer dependency than a dependency. Starting from a newer version of npm, it should automatically install peer dependency on `npm install`.

Also since that we're not directly importing from `react`, we don't need to make `react` as a peer dependency?

Reply:
Yeah, to be honest I just copied what we do in other packages. I agree that your proposal seems correct but I'd rather we do it in its own dedicated PR (probably after the npm upgrade too). cc @gziolo

Reply:
I agree that we need to rethink and document all the practices after we migrate Node and npm to the latest LTS version. For the time being, mirroring other packages is the best approach.